scripts/paper/generate_stwo_publication_bundle.sh

```
@@ -0,0 +1,213 @@
+  elapsed="$((ended_epoch - started_epoch))"
+  printf '%s\t%s\n' "$label" "$elapsed" >> "$BENCHMARKS"
+}
+
+cat > "$MANIFEST" <<MANIFEST
+generated_at_utc: $(date -u +"%Y-%m-%dT%H:%M:%SZ")
+repo_root: .
+git_commit: $(git rev-parse HEAD)
+git_commit_short: $(git rev-parse --short HEAD)
+git_branch: $(git rev-parse --abbrev-ref HEAD)
+rustc: $(rustup run "${NIGHTLY_TOOLCHAIN#+}" rustc --version)
+cargo: $(cargo $NIGHTLY_TOOLCHAIN --version)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)